---
title: "Mol Epi of SARS-CoV-2 in Cape Town, South Africa"
authors: "Eduan Wilkinson, Houriiyah Tegally, Tulio de Oliveira, Wolfgang Preiser, Gert Van Zyl & Susan Engelbrecht."
authorLinks: "https://www.krisp.org.za/ngs-sa/"
affiliations: "Division of Medical Virology, Department of Pathology, Faculty of Health and Medical Science, Stellenbosch University, Tygerberg, South Africa."
date: "July, 3 2020"
dataset: "https://nextstrain.org/community/EduanWilkinson/ncov_cpt"
abstract: "The Network for Genomics Surveillance in South Africa (NGS-SA) presents a molecular epidemiological study of the first 46 SARS-CoV-2 whole genomes from Cape Town, South Africa, during the first month of the epidemic."
---

# [Executive summary](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?c=clade_membership&d=map&p=full&transmissions=hide)
The emergence of a novel coronavirus, SARS-CoV-2, in December 2019, progressed to become a global pandemic in a few months and reached South Africa at the beginning of March 2020.
<br><br>
The first reported case was documented in the KwaZulu-Natal province on the east coast of the country. The case was associated with a retuning traveler from Europe. Within days similar cases of SARS-CoV-2 were being documented in Gauteng and the Western Cape Province.
<br><br>This report focuses on the early epidemic in Cape Town, which is the main economic and population hub in the Western Cape province of South Africa. To this end we have excluded sequences from KRISP - which was the focus of the [first NGS-SA report](https://nextstrain.org/groups/ngs-sa/narratives/COVID19-KZN-27June20).
<br><br>
The [Network for Genomics Surveillance in South Africa (NGS-SA)](https://www.krisp.org.za/ngs-sa/) was establish to investigate the genetic diversity of SARS-CoV-2 within the Republic. NGS-SA is a collaborative network of ten 
government and university laboratories in South Africa.  
	
<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

The network's is guided by six key principles:
<br><br>
* Sequence close to the site of sample collection.
* Being platform agnostic.
* Supporting local relevant public health priorities.
* Rapid and responsible opend data sharing.
* Creating a bioinformatics system to process and analyze data locally.
* Producing timely reports for poblicy makers to inform the public health response.

```
# [The Epidemic in South Africa has been very heterogeneous](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?c=clade_membership&d=map&p=full&transmissions=hide)
The South African epidemic has been very heterogeneous. For example, the first cases and deaths happened in KwaZulu-Natal (KZN) and Gauteng (GP). This was more pronounced in KZN, as a large nosocomial outbreak in a hospital in Durban resulted in KZN having the highest number of deaths until the Western Cape (WC) overtook it on 21 April 2020.<br><br> GP, province with the largest metropolitan area in the country, namely Johannesburg, had an unusual epidemic, as the majority of the initial cases were in middle aged and well-off individuals who had returned from holidays overseas. This translated in a very small number of deaths over time and infections were concentrated in the suburb of Sandton in Johannesburg. <br><br>However, the epidemic seems to be expanding the fastest in the Western Cape, especially in Cape Town. At the time of writing this report, this province has over 60% of all of the cases and deaths in South Africa.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
![alt text](https://www.krisp.org.za/imagesBIO/mapnarrative.png "Logo Title Text 1")


```

# [SARS-CoV-2 in Cape Town ](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?c=clade_membership&d=tree,map&l=radial&p=grid&transmissions=hide)
Our phylogenetic reconstruction contains 46 whole genomes sampled during the first month of the epidemic in Cape Town, South Africa. This roughly correlates to 10% of the diagnosed cases over the same time period. By analysing these genotypes against a backdrop of SARS-CoV-2 genotypes from around the world we can gain unique insights into the epidemic.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown

Some of the questions we are interested in are: 
<br><br>
* Is the epidemic in Cape Town caused by a single introductory event or was it seeded by multiple introductions? We hypothesis that the epidemic in Cape Town was seeded by multiple introductions of SARS-CoV-2.
* What is the source of the introduction(s)? We hypothesis that most of the viral introductions occured from European sources, given the high travel volumes between Cape Town and the European continent, particularly in the months of February and March.
* What is the timing of these viral introductions? Given the diangostic testing data we hypothesis that all the introductions occured between mid February and mid March, prior to travel restrictions. 
* Do we observe any unqiue mutations common to the viral population in Cape Town? 

```

# [Early genotypes from Cape Town ](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?c=clade_membership&d=tree,map&f_author=Susan%20Engelbrecht%20et%20al&l=radial&p=grid)
The phylogenetic reconstruction suggest at least nine introductions of SARS-CoV-2 into Cape Town. Accurate inference is hampered by the highly conserved nature of this virus. Given that only 10% of diagnosed cases were genotyped and analysed, the true number of viral introductions into Cape Town is most definitely much higher.<br><br>The observed genetic variants of SARS-CoV-2 in Cape Town can be divided into three main clades of the novel coronavirus: 19A, 20A and 20B.  

# [Timing of the introductions](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?d=tree,map&f_author=Susan%20Engelbrecht%20et%20al&l=radial&p=grid)
Based on the molecular clock analyses we can confirm that all the introductions into Cape Town occurred between the last week of February and in the first two weeks of March 2020. <br><br>February and March is high tourist season in Cape Town with hundereds of thousands of arrivals in the two months. Following the rising number of confirmed diagnosed cases in the Republic, the national government quickly enforced a lockdown. This basically brought international travel to and from South Africa to a halt. We therefore, don't expect any subsequent introductions into the Cape Metro area after 15 March 2020.<br><br>As travel restrictions are eased we will keep a close eye out for the emergence of new variants which may be the result of new introductions. 

# [Introduction of Clade-19A](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?d=tree,map&f_author=Susan%20Engelbrecht%20et%20al&p=full)
Three isolates (Tygerberg 4, 6 & 31) contained mutations more similar to the Asian variants of SARS-CoV-2, clustering in the 19A clade of SARS-CoV-2.<br><br>The genetic sequence of Tygerberg 4 does not differ to the Wuhan-Hu-1 reference strain. The Wuhan-Hu-1 reference was the first genome sequence of SARS-CoV-2 to be published in early January 2020 and is the genome against which all sequences are measured. Contact tracing points to a recent travel history to the United Kingdom for the individual from whom this virus were genotyped. This suggest transmission from China to the UK and onwards to Cape Town.<br><br> With regards to Tygerberg 6 and 31 these two genotypes share two common mutations (G26144T and C14805T) relative to the Wuhan-Hu-1 reference stain. For Tygerberg 6, the molecular clock and ancestral state reconstruction suggest spread from China to the United Kingdom and onwards to Cape Town. Along the chain of transmission additional mutation events occurred leading to the genetic profile we observe in sample 6.<br><br>With regards to Tygerberg 31, the most likely route of transmission for this genetic variant goes from China to Australia, the United States (defined by the T17247C mutation) and onwards to Cape Town. 

# [Introduction of Clade-20A](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?d=tree,map&f_author=Susan%20Engelbrecht%20et%20al&p=full)
The remainder of the Cape Town sequences were classified into clades 20A and 20B of SARS-CoV-2. The 20A-20C clades of SARS-CoV-2 is mainly associated with infections outside of Asia.<br><br>The 20A clade contains 28 genotypes from Cape Town. The result of two unique introductions. The reconstruction suggest transmission of a Spanish variant (A20268G) to the UK and onwards to Cape Town giving rise to the genetic profile of Tygerberg 2 & 3. Contact tracing confirms these two individuals to be contacts of a traveler who recently returned from the United Kingdom.

# [Clustered  Outbreak of 20A in Cape Town]()
The second 20A introduction led to a large clustered outbreak of SARS-CoV-2 in a busy supermarket in Cape Town. Contact tracing suggest Tygerberg 5 to be infected with a Dutch variant of SARS-CoV-2 while in the Netherlands.<br><br>The Dutch variant is defined by the common mutation A24862G relative to the Wuhan reference strain. From the Netherlands, this variant spread to several countries (e.g. New Zealand, Colombia & Austria). This cluster also contained another sequences from South Africa (R07601) sampled in the northern province of Limpopo.

# [Mutation A5209G]()
Following the introduction of the Dutch variant into Cape Town, the virus further diversified with the accumulation of the mutation A5209G before causing a large out in the supermarket.<br><br>This variant is only observed in the supermarket outbreak in Cape Town and no where else on earth. Keeping tack of mutations such as this one, will allow us to understand how the virus spread between different locations within the city, the surrounding country side and throughout the Republic. 

# [Introduction of 20B](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?d=tree,map&f_author=Susan%20Engelbrecht%20et%20al&p=full)
Fifteen of our Cape Town sequences clustered in clade 20B. We believe these fifteen sequences to be the result of at least 4 unique introductions. Sample 29 and 43 appear to be unique introductions from the UK and the Netherlands, respectively.<br><br>Tygerberg 30 and 32 appear to be due to a unique introduction of a UK variant (defined by the common mutation G10097A).<br><br>The remaining eleven Tygerberg sequences cluster intermittently in a clade rooted in Italy. The mutation profile of these eleven sequences are so close to one another that its difficult to distinguish whether they are the result of a single introduction or due to multiple introductions into Cape Town. Given their close genetic similarity we chose to define it as a single introduction.   

# [Evolutionary rate of SARS-CoV-2](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?d=tree&f_author=Susan%20Engelbrecht%20et%20al&l=clock&p=full)
Based on the full phylogenetic tree, we estimated a mean mutation rate of ~25 substitutions per genome per year. For a virus with a genome of ~30kbp, this roughly translates to a mutation rate of 0.0008 sub/site/year.<br><br>To put this into context, SARS-CoV-2 and HIV are both RNA viruses. That means both lack the proof reading ability when copying their genetic material. However, HIV evolves at a much faster rate of 0.004 sub/site/year or ~16x faster than SARS-COV-2. 

<!-- This is left-side text -->
# [Scientific credit](https://nextstrain.org/community/EduanWilkinson/ncov_cpt?c=submitting_lab&d=tree&p=full)

These data were shared via [GISAID](https://gisaid.org).
We gratefully acknowledge their contributions.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
The SARS-CoV-2 genomes were generously shared by scientists at these submitting labs:

* African Centre of Excellence for Genomics of Infectious Diseases (ACEGID), Redeemer's University, Ede, Osun State, Nigeria
* Albertsen lab, Department of Chemistry and Bioscience, Aalborg University, Denmark
* Andersen lab at Scripps Research
* Area de Secuenciacion del Laboratorio de Virologia del Hospital de Ninos Dr. Ricardo Gutierrez
* ARGO Open Lab Platform for Genome sequencing
* BCCDC Public Health Laboratory
* Beijing Genomics Institute (BGI)
* Beijing Institute of Microbiology and Epidemiology
* Bergthaler laboratory, CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences
* BGI & Institute of Microbiology, Chinese Academy of Sciences & Shandong First Medical University & Shandong Academy of Medical Sciences & General Hospital of Central Theater Command of People's Liberation Army of China
* BGI-shenzhen & The First Affiliated Hospital of Guangzhou Medical University
* Bioinformatics Laboratory - LNCC
* Bioinformatics Laboratory / LNCC
* Bioinformatics Research Group, Szentagothai Research Centre
* Bureau of Laboratories
* Center for Global Health, University of New Mexico Health Sciences Center
* Center of Medical Microbiology, Virology, and Hospital Hygiene, University of Duesseldorf
* Center of Scientific Excellence for Influenza Viruses, National Research Centre (NRC), Egypt.
* Center of Scientific Excellence for Influenza Viruses,National Research Centre (NRC), Egypt.
* Centers for Disease Control and Prevention, Dengue Branch
* Centers for Disease Control, R.O.C. (Taiwan)
* Centre for Dengue Research
* Chan-Zuckerberg Biohub
* Charite Universitatsmedizin Berlin, Institute of Virology
* Chinese PLA Institute for Disease Control and Prevention
* Chiu Laboratory, University of California, San Francisco
* Chongqing Municipal Center for Disease Control and Prevention
* Clinical Laboratory
* CNR Virus des Infections Respiratoires - France SUD
* Collaboration between the University of Melbourne at The Peter Doherty Institute for Infection and Immunity, and the Victorian Infectious Disease Reference Laboratory
* COVID-19 Genomics UK (COG-UK) Consortium
* COVID-19 Network Investigations (CONI) Alliance
* Dasman Diabetes Institute
* deCODE genetics
* Department for Virology, Molecular Biology and Genome Research, R. G. Lugar Center for Public Health Research,  National Center for Disease Control and Public Health (NCDC) of Georgia.
* Department of Clinical Laboratory, the First People's Hospital of Yunnan Province
* Department of Health Technology and Informatics, Faculty of Health and Social Science, The Hong Kong Polytechnic University
* Department of Healthcare Biotechnology
* Department of Infection, Immunity and Cardiovascular Disease, The Florey Institute, The Medical School, University of Sheffield
* Department of Laboratory Medicine, Lin-Kou Chang Gung Memorial Hospital, Taoyuan, Taiwan
* Department of Laboratory Medicine, Lin-Kou Chang Gung Memorial Hospital, Taoyuan, Taiwan.
* Department of Medical Microbiology
* Department of Medical Microbiology, Faculty of Medicine, University of Malaya
* Department of Microbiology, Faculty of Medicine, Chinese University of Hong Kong, Hong Kong SAR, China
* Department of Microbiology, Gandhi Medical College and Hospital, Secendrabad, Hyderabad, India
* Department of Microbiology, Institute for Viral Diseases, College of Medicine, Korea University
* Department of Microbiology, PathWest QEII Medical Centre
* Department of Microbiology, Zhejiang Provincial Center for Disease Control and Prevention
* Department of Neurovirology, National Institute of Mental Health and Neuroscience (NIMHANS)
* Department of Pathology and Medicine, New York University School of Medicine
* Department of Virology, Faculty of Medicine, University of Helsinki, Helsinki, Finland
* Department of Virology, University of Helsinki and Helsinki University Hospital, Helsinki, Finland
* Departments of Pathology and Medicine, New York University School of Medicine
* Diagnostic Virology Laboratory, United States Department of Agriculture, National Veterinary Services Laboratories
* Division of Consolidated Laboratories
* Division of Consolidated Laboratory Services
* Division of Viral Diseases, Center for Laboratory Control of Infectious Diseases, Korea Centers for Diseases Control and Prevention
* Division of Viral Diseases, Centers for Disease Control and Prevention
* Erasmus Medical Center
* Fujian Center for Disease Control and Prevention
* Genomic Laboratory (GLAB), Istanbul Technical University
* GIGA Medical Genomics
* Grubaugh Lab - Yale School of Public Health
* Guangdong Provincial Center for Diseases Control and Prevention
* Guangdong Provincial Institution of Public Health
* Hangzhou Center for Disease Control and Prevention
* Hangzhou Center for Diseases Control and Prevention
* Hellenic Pasteur Institute, Public Health Laboratories, Unit of Bioinformatics and Applied Genomics
* Hong Kong Department of Health
* Hopitaux universitaires de Geneve Laboratoire de Virologie
* Hospital Universitario 12 de Octubre
* Hospital Universitario La Paz
* Houston Methodist Hospital
* Hubei Provincial Center for Disease Control and Prevention
* Human Genetic Research Center
* Inciensa, Instituto Costarricense de Investigacion y Ensenanza en Nutricion y Salud
* Indian Council of Medical Research-National Institute of Virology, Microbial Containment Complex
* Infectious Disease Research Department, King Abdullah International Medical Research Center (KAIMRC)
* Inspection Center of Hangzhou Center for Disease Control and Prevention
* Institut Pasteur de Dakar
* Institute for Medical Research Infectious Disease Research Centre,  National Institutes of Health,  Ministry of Health Malaysia
* Institute for Medical Research, Infectious Disease Research Centre, National Institutes of Health, Ministry of Health Malaysia
* Institute of Applied Biotechnologies a.s.
* Institute of Environmental Science and Research (ESR)
* Institute of Medical Virology, University of Zurich
* Institute of Microbiology and Immunology, Faculty of Medicine, University of Ljubljana
* Institute of Microbiology Universidad San Francisco de Quito
* Institute of Microbiology, Universidad San Francisco de Quito
* Institute of Pathogen Biology, Chinese Academy of Medical Sciences & Peking Union Medical College
* Institute of Tropical Medicine
* Institute of Virology, Biomedical Research Center of the Slovak Academy of Sciences, Bratislava; Comenius University Science Park, Bratislava
* Instituto Adolfo Lutz Interdisciplinary Procedures Center Strategic Laboratory
* Instituto Adolfo Lutz, Interdiciplinary Procedures Center, Strategic Laboratory
* Instituto Adolfo Lutz, Interdisciplinary Procedures Center, Strategic Laboratory
* Instituto de Diagnostico y Referencia Epidemiologicos
* Instituto de Diagnostico y Referencia Epidemiologicos (INDRE)
* Instituto de Salud Carlos III
* Instituto de Salud Publica de Chile
* Instituto Gulbenkian de Ciencia
* Instituto Nacional de Ciencias Medicas y Nutricion
* Instituto Nacional de Ciencias Medicas y Nutricion Salvador Zubiran
* Instituto Nacional de Enfermedades Respiratorias
* Instituto Nacional de Salud Universidad Cooperativa de Colombia Instituto Alexander von Humboldt Imperial College-London London School of Hygiene & Tropical Medicine
* Instituto Nacional de Saude (INSA)
* Instituto Nacional Enfermedades Infecciosas C.G.Malbran
* Instituto Octavio Magalhaes / Fundacao Ezequiel Dias (IOM/Funed)
* Instituto Oswaldo Cruz FIOCRUZ - Laboratory of Respiratory Viruses and Measles (LVRS)
* Israel Institute for Biological Research
* Istituto Zooprofilattico Sperimentale dell'Abruzzo e Molise ""G. Caporale""
* Istituto Zooprofilattico Sperimentale dellAbruzzo e Molise G. Caporale
* Jiangsu Provincial Center for Disease Control & Prevention
* Jiangxi Province Center for Disease Control and Prevention
* Kabara Cancer Research Institute
* Korea Centers for Disease Control & Prevention (KCDC) Center for Laboratory Control of Infectious Diseases Division of Viral Diseases
* KRISP, Africa Research Innovation and Sequencing Platform
* KU Leuven, Clinical and Epidemiological Virology
* Laboratoire de virologie - Ecole Nationale Veterinaire de Toulouse
* Laboratoire National de Sante, Microbiology, Epidemiology and Microbial Genomics
* Laboratoire Nationale de Sante, Microbiology, Epidemiology and Microbial Genomics
* Laboratoire Virpath, CIRI U111, UCBL1, INSERM, CNRS, ENS Lyon
* Laboratory for Functional Genome Analysis, Dept. Genomics, Gene Center of the LMU Munich
* Laboratory of Applied Genetics
* Laboratory of Biology, Department of Medicine, Democritus University of Thrace
* Laboratory of Biology, Department of Medicine, Democritus University of Thrace, Greece
* Laboratory of Infectious Diseases, Department of Biomedical and Clinical Sciences L. Sacco, University of Milan
* Laboratory of Molecular Genetics, 2nd Faculty of Medicine, Charles University in Prague, Prague, Czech Republic
* Laboratory of Recombinant Vaccines
* Laboratory of Virology, INMI Lazzaro Spallanzani IRCCS
* Latvian Biomedical Research and Study Centre
* Malaysia Genome Institute
* Mason Lab
* Microbial Genome Sequencing Center
* Microbial Genomics Core Lab, National Taiwan University Centers of Genomic and Precision Medicine
* Microbial Genomics Laboratory, Institut Pasteur Montevideo
* Microbial Genomics Laboratory, Institut Pasteur Montevideo, Uruguay
* Microbiological Diagnostic Unit Public Health Laboratory
* Microbiological Diagnostic Unit Public Health Laboratory and Victorian Infectious Diseases Reference Laboratory, Doherty Institute
* Microbiological Diagnostic Unit Public Health Laboratory and Victorian Infectious Diseases Reference Laboratory, The Peter Doherty Institute for Infection and Immunity
* Microbiology
* Microbiology and Immunology department
* Ministry of Health Turkey
* Minnesota Department of Health, Public Health Laboratory
* Molecular Biology and Biotechnology Lab II
* MRCG at LSHTM Genomics lab
* MSHS Pathogen Surveillance Program
* National Centre for Infectious Diseases, National Centre for Infectious Diseases
* National Influenza Center - National Institute of Hygiene and Epidemiology (NIHE)
* National Influenza Center, Indian Council of Medical Research - National Institute of Virology
* National Influenza Center, Indian Council of Medical Research-National Institute of Virology
* National Influenza Center, National Institute of Hygiene and Epidemiology (NIHE)
* National Institute for Communicable Disease Control and Prevention (ICDC) Chinese Center for Disease Control and Prevention (China CDC)
* National Institute for Viral Disease Control and Prevention, China CDC
* National Institute for Viral Disease Control and Prevention, China CDC,  Yunnan Provincial CDC
* National Institute of Health. Department of medical Sciences, Ministry of Public Health, Thailand
* National Microbiology Laboratory
* National Public Health Laboratory
* National Public Health Laboratory, National Centre for Infectious Diseases
* National Reference Center for Viruses of Respiratory Infections, Institut Pasteur, Paris
* National Research Center for Translational Medicine (Shanghai), Ruijin Hospital affiliated to Shanghai Jiao Tong University School of Medicine & Shanghai Public Health Clinical Center
* Norwegian Institute of Public Health
* Norwegian Institute of Public Health, Department of Virology
* NSW Health Pathology - Institute of Clinical Pathology and Medical Research; Centre for Infectious Diseases and Microbiology Laboratory Services; Westmead Hospital; University of Sydney
* NSW Health Pathology - Institute of Clinical Pathology and Medical Research; Westmead Hospital; University of Sydney
* Onderzoeksgroep Virologie
* Ontario Agency for Health Protection and Promotion (OAHPP)
* Pathogen Discovery, Respiratory Viruses Branch, Division of Viral Diseases, Centers for Disease Control and Prevention
* Pathogen Genomics Center, National Institute of Infectious Diseases
* Pathogen Sequencing Lab, National Institute for Biomedical Research (INRB)
* Pathogenic Microorganisms Variability Laboratory
* PathWest Laboratory Medicine WA
* Philippine Genome Center
* Philippine Genome Center, University of the Philippines System
* Programme in Emerging Infectious Diseases, Duke-NUS Medical School
* Public Health Laboratory, Saudi CDC
* Public Health Ontario Laboratories
* Public Health Virology Laboratory
* Public Health Wales Microbiology Cardiff
* R. G. Lugar Center for Public Health Research,  National Center for Disease Control and Public Health (NCDC) of Georgia.
* Research Institute for Tropical Medicine
* Respiratory Virus Unit, Microbiology Services Colindale, Public Health England
* Saitama Medical University
* School of Public Health, The University of Hon g Kong
* School of Public Health, The University of Hong Kong
* Seattle Flu Study
* Sequencing and Bioinformatics Service and Molecular Epidemiology Research Group. FISABIO-Public Health
* Sequencing and Bioinformatics Service and Molecular Epidemiology Research Group. FISABIO-Public Health.
* State Key Laboratory for Diagnosis and Treatment of Infectious Diseases, National Clinical Research Center for Infectious Diseases, First Affiliated Hospital, Zhejiang University School of Medicine, Hangzhou, China 310003
* State Key Laboratory of Virology, Wuhan University
* State Research Center of Virology and Biotechnology VECTOR, Department of Collection of Microorganisms
* State Veterinary Institute Prague
* Statens Serum Institute
* Swiss National Reference Centre for Influenza
* Taiwan Centers for Disease Control
* Takayuki Hishiki Kanagawa Prefectural Institute of Public Health
* Takayuki Hishiki Kanagawa Prefectural Institute of Public Health, Department of Microbiology
* Technology Centre, Guangzhou Customs
* TGen North
* The National Institute of Public Health Center for Epidemiology and Microbiology
* The National Laboratory of Health, Environment and Food, Maribor, Slovenia
* The Ohio State University-James Molecular Lab at Polaris
* The Public Health Agency of Sweden
* The University of Hong Kong
* TSGH-CP molecular lab
* UCD National Virus Reference Laboratory
* Unit for Laboratory Development and Technology Transfer, Public Health Agency of Sweden
* University Hospital Basel, Clinical Bacteriology
* University Hospitals of Geneva Laboratory of Virology
* University of Otago
* University of Wisconsin Madison, AIDS Vaccine Research Laboratories
* University of Wisconsin-Madison AIDS Vaccine Research Laboratories
* University of Wisconsin-Madison, AIDS Vaccine Research Laboratories
* Utah Public Health Laboratory
* UW Virology Lab
* Victorian Infectious Diseases Reference Laboratory and Microbiological Diagnostic Unit Public Health Laboratory, Doherty Institute
* ViFU
* ViroGenetics - BSL3 Laboratory of Virology; Human Genome Variation Research Group & Genomics Centre MCB; Bioinformatics Research Group  Department of Virology
* Virology Department, Royal Infirmary of Edinburgh, NHS Lothian
* Virology Laboratory, Scientific Department, Army Medical Center
* Virology Unit, Institut Pasteur du Cambodge (Sequencing done by: Jessica E Manning/Jennifer A Bohl at Malaria and Vector Research Research Laboratory, National Institute of Allergy and Infectious Diseases and Vida Ahyong from Chan-Zuckerberg Biohub)
* WACCBIP, University of Ghana
* Wadsworth Center, New York State Department.of Health
* WHO National Influenza Centre Russian Federation
* Wuhan Institute of Virology, Chinese Academy of Sciences
```
